.travis.yml

```diff
@@ -30,7 +30,7 @@ services:
 before_install:
   - if [[ $TRAVIS_PHP_VERSION != 7.1 ]] ; then phpenv config-rm xdebug.ini; fi
   - echo "extension = memcached.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - pecl install -f redis
```

Comment:
@lucasmichot you're right, but the problem is the installed version. for PHP 7 and 7.1, it's ok, but for PHP 5.6, it has an old version of phpredis extension (< 2.2.7).